equipment-models/src/main/java/com/telecominfraproject/wlan/equipment/models/ElementRadioConfiguration.java

```diff
@@ -58,7 +58,7 @@ public static ElementRadioConfiguration createWithDefaults(RadioType radioType)
 			returnValue.setBackupChannelNumber(44);  
 		} else if (radioType == RadioType.is5GHzU) {
 			returnValue.setChannelNumber(149);
-			returnValue.setBackupChannelNumber(154); 
+			returnValue.setBackupChannelNumber(153);
```

Comment:
I think 157 would be a better choice. The goal is to try and pick the primary and backup so that they're in different bands. eg for a 5G channel, 36 and 153 are in different bands for 20MHz, 40MHz and 80MHz channel bandwidths. For 5G U/L, the best we can do is choose channels that are in different bands for 20 and 40 MHz. The 5GHzL defaults of 36 & 44 are in different bands for 20 and 40MHz. However 5GHzU 149 and 153 are in the same band at 40MHz. If we default 5GHzU to 149 and 157 then they'll be in different bands for 20 and 40 MHz.